shared/extension/Configuration.hpp

```diff
@@ -294,24 +300,21 @@ namespace NUClear::dsl {
                         const auto* binary = basename(data.data());
 
                         // Get relative path to config file.
-                        auto components = utility::strutil::split(watch.path, '/');
-                        std::string relativePath("");
+                        const auto components = utility::strutil::split(watch.path, '/');
+                        fs::path relativePath{};
                         bool flag = false;
                         for (const auto& component : components) {
                             // Ignore the hostname/binary name if they are present.
                             if (flag && (component.compare(hostname) != 0) && (component.compare(binary) != 0)) {
-                                relativePath.append(component + "/");
+                                relativePath = relativePath / component;
                             }
 
                             // We want out paths relative to the config folder.
-                            if (component.compare("config") == 0) {
+                            if (component.compare(fs::path("config")) == 0) {
```

Comment:
Why are we wrapping config in `fs::path` here?

Reply:
`ctrl + f` took the wheel and added one too many here :joy: 
Thank ya sir